spec/src/convert-question-spec.ls

```diff
@@ -545,4 +545,7 @@ describe 'parameters' ->
     for appearance in [ 'Slider', 'Vertical Slider', 'Picker' ]
       result = { type: \inputNumeric, appearance, selectRange: { min: 13, max: 42 }, selectStep: 1.5 } |> convert-simple
       expect(result.parameters).toBe('start=13 end=42 step=1.5')
-
+    # Slider parameters are quoted in XForm but must be numeric in XLSForm
```

Comment:
This new test shows the intended added behaviour: string values get cast to numeric.